demo/content/Form.js

```diff
@@ -38,8 +39,18 @@ export default class DemoContentForm extends Component {
       console.log(error) // eslint-disable-line no-console
    }
 
-   handleSubmit(data, files, formData) {
-      console.log(data, files) // eslint-disable-line no-console
+   handleSubmit(data, files, formData, constructFormData) {
+      /* eslint-disable */
+      console.log(data)
+      console.log(files)
+      const formdata = {}
+      for (const pair of formData.entries()) formdata[pair[0]] = pair[1]
```

Comment:
`FormData#entries` (and most other `FormData` methods) are not supported on many browsers, including IE Edge & Safari. See https://developer.mozilla.org/en/docs/Web/API/FormData#Browser_compatibility